src/plot/xy-plot.js

```diff
@@ -221,14 +221,14 @@ class XYPlot extends React.Component {
   }
 
   /**
-   * Get the list of scale-related settings that are optionally passed into x-y-plot
+   * Get the list of optional scale-related settings
    * @param {Object} props Object of props.
    * @returns {Object} Optional Props.
    * @private
    */
   _getOptionalScaleProps(props) {
```

Comment:
Yes or else attributes following `[name]Padding` schema will not be passed to getMissingScaleProps. Not all props are passed in only ones that fall under "defaultScaleProps, optionalScaleProps, userScaleProps" It is perhaps forward looking though and could be reduced to just checking if a property ends with Padding. I feel this would be less general though than checking against OPTIONAL_SCALE_PROPS